openai.text/client.bal

```diff
@@ -59,6 +59,7 @@ public isolated client class Client {
     # Creates a completion for the provided prompt and parameters
     #
     # + return - OK 
+    @display {label: "Create Completion"}
```

Comment:
Shouldn't we modify the client level annotation as well, to fix the connector name shown in choreo marketplace?

Reply:
I actually did those too but haven't pushed them. Please check now.